process-manager.js

```diff
@@ -88,38 +93,46 @@ class ProcessManager {
 	release(process) {
 		process.release();
 	}
+
 	send() {
 		if (!this.processes.length) {
 			return Promise.resolve(this.receive.apply(this, arguments));
 		}
 
+		// Prevents the arguments object from leaking.
+		let args = [];
+		for (let i = 0; i < arguments.length; i++) {
+			args.push(arguments[i]);
+		}
```

Comment:
I was talking more about accessing arguments values directly, not `.apply()`, which I think may potentially be safe? I was going off SO, which was likely a mistake. I'm going to need to experiment more to see why exactly what I did worked and how else I can improve this or get arguments to leak before I finish working on this pullreq.